package.json

```diff
@@ -8,7 +8,7 @@
     "build:less": "lessc --remcalc packages/oui/stylekit.less dist/oui.css",
     "build:less-min": "lessc --remcalc --clean-css packages/oui/stylekit.less dist/oui.min.css",
     "build:icons": "mkdirp dist/icons && npm run build:icons-generator && node scripts/generate-icons-indexes.js",
-    "build:icons-generator": "icon-font-generator packages/oui-icons/svg/*.svg -p oui-icon -o dist/icons -t span",
+    "build:icons-generator": "icon-font-generator packages/oui-icons/svg/*.svg -p oui-icon -o dist/icons -t span --types 'woff, woff2' --css false --html true",
```

Comment:
Why removing the CSS, the HTML demo is useless without it ?

Reply:
I will revert and keep the full doc (html +css)